### PR TITLE
added regional sts endpoints

### PR DESCRIPTION
**Pull request by arafato**

### CODE REVIEW

File: sdk/endpoints/endpoints_config.go [hunk 1]
```diff
@@ -2577,7 +2576,84 @@ const endpointsJson =`{
...
+				},
+				{
+					"region": "cn-beijing",
+					"endpoint": "sts.cn-beijing.aliyuncs.com"
+				},
+				{
+					"region": "cn-qingdao",
+					"endpoint": "sts.cn-qingdao.aliyuncs.com"
+				},
+				{
```
Comment (JacksonTian) [inline sdk/endpoints/endpoints_config.go]: Hi  the region list is incomplete.

Comment (JacksonTian) [reply sdk/endpoints/endpoints_config.go]: I will trigger STS release a new version SDK to enable endpoint rule.

Comment (arafato) [reply sdk/endpoints/endpoints_config.go]: I've put all regions according to https://www.alibabacloud.com/help/doc-detail/66053.htm

Comment (arafato) [reply sdk/endpoints/endpoints_config.go]: @JacksonTian so when will this be merged or updated? No updates so far?

Comment (wenzuochao) [reply sdk/endpoints/endpoints_config.go:2652]: You can pull the latest sts sdk. The problem has been solved.

**Pull request closed without merging**